sdk/src/main/java/io/opentdf/platform/sdk/Config.java

```diff
@@ -99,12 +97,14 @@ public static class TDFReaderConfig {
         // Optional Map of Assertion Verification Keys
         AssertionVerificationKeys assertionVerificationKeys = new AssertionVerificationKeys();
         boolean disableAssertionVerification;
+        KeyType sessionKeyType;
     }
 
     @SafeVarargs
     public static TDFReaderConfig newTDFReaderConfig(Consumer<TDFReaderConfig>... options) {
         TDFReaderConfig config = new TDFReaderConfig();
         config.disableAssertionVerification = false;
+        config.sessionKeyType = KeyType.RSA2048Key;
```

Comment:
I assume the user knows the default is `KeyType.RSA2048Key` for the session and wrapping key.